docs/source/how-to-upstream.mdx

```diff
@@ -116,10 +147,28 @@ huggingface-cli lfs-enable-largefiles
 
 You should install this for each model repository that contains a model file. Once installed, you are now able to push files larger than 5GB.
 
-## Managing files in a repo without Git with the `create_commit` API
+## Upload without Git installed
 
-`huggingface_hub` also offers a way to upload files to the Hub without Git installed on your system with the [`create_commit`] method of [`HfApi`].
-For example, if you want to upload two files and delete another file in a Hub repo:
+<Tip>
+
+The [`create_commit`] is really useful for certain use-cases. In general though, it is simpler to use one of the other uploading functions mentioned above.
+
+</Tip>
+
+If you don't have Git installed on your system, use the [`create_commit`] function to upload your file. [`create_commit`] uses the HTTP protocol to upload files to the Hub. Large files and binary files are automatically uploaded with the Git LFS protocol.
```

Comment:
The upload_file and upload_folder functions also just use HTTP and don't require git. From their docs:

> The upload is done through a HTTP post request, and doesn't require git or git-lfs to be installed.

So I'd say here that create_commit is just a low-level API, in case you want to work at a commit level.